test/tests/functional/pbs_sched_preempt_enforce_resumption.py

```diff
@@ -125,7 +125,8 @@ def test_suspended_job_ded_time_calendared(self):
                     self.server.status(
                         JOB,
                         attr,
-                        id=jid1)[0][attr],
+                        id=jid1)
+                    [0][attr],
```

Comment:
This isn't what I meant.  Do each step in a different line.  Doing them all in one 4 layer call is confusing.